src/WebJobs.Script/Workers/Rpc/RpcWorkerChannel.cs

```diff
@@ -127,6 +132,47 @@ public async Task StartWorkerProcessAsync()
             await _workerInitTask.Task;
         }
 
+        public async Task<WorkerStatus> GetWorkerStatusAsync()
+        {
+            var workerStatus = new WorkerStatus();
+
+            if (!string.IsNullOrEmpty(_workerCapabilities.GetCapabilityState(RpcWorkerConstants.WorkerStatus)))
```

Comment:
We'll need to update all the workers to support this new capability, so it lights up